crates/holochain/src/conductor/space.rs

```diff
@@ -241,6 +241,13 @@ impl Spaces {
             }
         };
 
+        // If node_agents_in_spaces is not yet initialized, we can't know anything about
+        // which cells are blocked, so avoid the race condition by returning false
+        if cell_ids.is_empty() {
+            tracing::warn!(?target_id, "is_blocked(): No agents were found for the target. This indicates that preflight agent exchange may not be working properly.");
```

Comment:
exactly... shouldn't we fix the preflight if it's broken and causing problems?

Reply:
Do we know that Holochain recovers from this situation? Or is is possible that it stays in this state?